### ISSUE TITLE
语感建议：若其者  --> 若其阳者

### ISSUE THREAD

**Issue by lymslive** [LingDong-/wenyan-lang#513]
在读 stdlib 一些代码，感觉
“若其者”读之太不顺，建议
“若其阳者”表示 `if(most_recent_var)`
若其阴者”表示 `if(!most_recent_var)`

**Comment by LingDong-** [comment]
You're right! 若其阳者 reads more smoothly. I'll definitely consider adding this.

Notice that 若其等於陽者 若其等於陰者 is already supported and does (almost) exactly the same thing. `若其者` is just a shorthand and is arguably a stylistic choice. Only difference is `if(!a)` does a typecast from any type to bool, which, now that wenyan-lang is stricter on types, we need to consider how we want to handle this.

Thanks for bringing this up!

**Comment by cuixiping** [comment]
I like `若其然者 若其不然者`

**Comment by hustcer** [comment]
若其是者，若其非者？
